### PR TITLE
Add CrowtyCraft Network server

### PR DESCRIPTION
**Pull request by LucasitoInDev**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by LucasitoInDev** [comment]
I have corrected the links so that they are not "custom" as indicated by the regulations

Anything that you see as unnecessary or that there is an error, please let me know and I will correct it :D

File: minecraft_servers/crowtycraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      "color": "#4f24b5"
+    },
+    "buildbattle": {
+      "name": "BuildBattle",
+      "color": "#4f24b5"
+    }
+  },
+  "social": {
+    "web": "https://crowtycraft.es/",
+    "web_shop": "https://tienda.crowtycraft.es/",
```
Comment (JakobBruening) [inline minecraft_servers/crowtycraft/manifest.json:39]: Remove "/" at the end

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by LucasitoInDev** [comment]
Done!

File: minecraft_servers/crowtycraft/manifest.json (hunk 1 above)
Comment (LucasitoInDev) [reply minecraft_servers/crowtycraft/manifest.json]: Done!

**Review by JakobBruening — APPROVED** [review]
Thank you! :)

**Pull request merged**